Source/WebCore/rendering/svg/RenderSVGHiddenContainer.cpp

```diff
@@ -45,15 +45,6 @@ void RenderSVGHiddenContainer::layout()
     clearNeedsLayout();    
 }
 
-void RenderSVGHiddenContainer::styleDidChange(StyleDifference diff, const RenderStyle* oldStyle)
-{
-    RenderSVGContainer::styleDidChange(diff, oldStyle);
-
-    // Ensure that descendants with layers are rooted within our layer.
-    if (hasLayer())
-        layer()->setIsOpportunisticStackingContext(true);
```

Comment:
This has implications (not sure if Web exposed though). I originally intended not to mark a layer as a CSS stacking context, if it is not from CSS point of view. I didn't come up with the opportunistic stacking context logic, but re-used it. Originall written by Simon IIRC for a different purpose, but no longer used now except here in LBSE.

Just saying: we need to be careful, if this is Web exposed, and query-able. (should a <defs> establish a checking context? spec refs would be helpful).